src/pytorch_lightning/loops/fit_loop.py

```diff
@@ -174,15 +174,9 @@ def done(self) -> bool:
             met_min_epochs = self.epoch_progress.current.processed >= self.min_epochs if self.min_epochs else True
             met_min_steps = self.epoch_loop.global_step >= self.min_steps if self.min_steps else True
             if met_min_epochs and met_min_steps:
-                self.trainer.should_stop = True
```

Comment:
Shouldn't this stay?

`should_stop = met_min_epochs and met_min_steps`

Reply:
it's already `True`. L-172